crates/ruff/src/rules/pydocstyle/helpers.rs

```diff
@@ -97,3 +98,25 @@ pub fn should_ignore_definition(
     }
     false
 }
+
+/// Returns `true` if a method is defined as one of supplied decorators in
+/// `property_decorators`.
+pub fn is_property(
+    checker: &Checker,
+    decorator_list: &[Expr],
+    property_decorators: &BTreeSet<String>,
+) -> bool {
+    if property_decorators.is_empty() {
+        return false;
+    }
+
+    decorator_list.iter().any(|expr| {
+        checker
+            .resolve_call_path(map_callable(expr))
+            .map_or(false, |call_path| {
+                property_decorators
+                    .iter()
+                    .any(|decorator| to_call_path(decorator) == call_path)
+            })
+    })
+}
```

Comment:
A portion of ruff::ast::function_type::classify is basically the same snippet. Would it make sense to make this a helper somewhere higher than in ruff::rules::pydocstyle::helpers?

Reply:
(also, function name and description look bad to me, but I can't name them better - language barrier -, any suggestions on improving them?)

Reply:
Just looking at some of the existing conventions... one sec...

Reply:
Not relevant not that the function is gone :)